ieducar/Reports/StudentSheetReport.php

```diff
@@ -626,7 +626,7 @@ public function getSqlMainReport()
                          ELSE 'UTILIZA'
                      END) AS transporte_aluno,
 
-  (SELECT to_char(data_cadastro, 'dd/MM/yyyy')
+  (SELECT to_char(data_matricula, 'dd/MM/yyyy')
```

Comment:
Boa, @ralphg6, apenas sugiro que use um `COALESCE`, pois `data_matricula` pode ser NULL em alguns cenários.

```suggestion
  (SELECT to_char(COALESCE(data_matricula, data_cadastro), 'dd/MM/yyyy')
```